### PR TITLE
Englishization/animationClip

### LINKED ISSUE #201: 【Englishization】动画切片

### PR DESCRIPTION
**Pull request by luzhuang**
Englishization/animationClip
Close #201

### CODE REVIEW

File: docs/artist-animation-clip.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+order: 2
+
+title: 动画切片
+
+type: 美术
+---
+
+The AnimationClip(**AnimationClip**) is **a combination of animations on a time axis**, which can be the rotation, transformation, scaling, weight animation of multiple objects, such as **walking, running, jumping** can be exported separately 3 AnimaionClips; Oasis engine can choose which AnimaionClip to play, provided that the FBX or glTF exported by the modeling software contains multiple AnimaionClips.
```
Comment (yangfengzzz) [inline docs/artist-animation-clip.md:9]: AnimationClips

File: docs/artist-animation-clip.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+3. Double-click the plugin and click the **Import** button:
+   
+![image.png](https://gw.alipayobjects.com/zos/OasisHub/a44674c8-b105-4bfe-b128-46f4685a9758/1622551409520-2797ff27-65e9-4360-aa67-6d8438ec46f7.png)
+
+If the installation is successful, you can see an additional **AntG** option in the menu bar:
+
+![image.png](https://gw.alipayobjects.com/zos/OasisHub/aca2c330-4b8b-44ca-b641-f245b8667e96/1622551587689-1f963ad1-2530-4d5a-b312-25a87e7b99e0.png)
+
+4. Drag and drop the FBX file to be sliced into the resource pannel:
```
Comment (yangfengzzz) [inline docs/artist-animation-clip.md:69]: panel

File: docs/artist-animation-clip.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+![image.png](https://gw.alipayobjects.com/zos/OasisHub/b7d9d6b0-cd94-4151-b4c4-06b4102bd656/1622552692349-5551e817-70b5-4093-9b40-b9a7dd45c365.png)
+
+6. At this point, the AnimationClip resource has been completed. Next, we will introduce how to export. first drag the resource to the node tree:
+![image.png](https://gw.alipayobjects.com/zos/OasisHub/5a82bdcb-b1c2-4dfd-942d-85cf441958bd/1622552417304-8b1f1b7b-d99f-47d7-925f-5a70468d4a3e.png)
+
+7. Then add **Animator** Component to the node:
+
+![image.png](https://gw.alipayobjects.com/zos/OasisHub/34102994-c037-4ad7-be0a-941c24f1347f/1622552470594-9e7df115-24c6-4a16-9a64-a7c28206900e.png)
+
+8. As you can see, the Animator component needs to bind an Animator Controller resource, so we need to create a new Animator Controller resource in the resource pannel:
```
Comment (yangfengzzz) [inline docs/artist-animation-clip.md:86]: panel

**Review by yangfengzzz — CHANGES_REQUESTED** [review]

File: docs/artist-animation-clip.md (hunk 1 above)
Comment (luzhuang) [reply docs/artist-animation-clip.md:7]: fixed

File: docs/artist-animation-clip.md (hunk 2 above)
Comment (luzhuang) [reply docs/artist-animation-clip.md]: fixed

File: docs/artist-animation-clip.md (hunk 3 above)
Comment (luzhuang) [reply docs/artist-animation-clip.md]: fixed

File: docs/artist-animation-clip.md (hunk 1 above)
Comment (yangfengzzz) [reply docs/artist-animation-clip.md:7]: still left three AnimaionClips to fix

**Review by yangfengzzz — APPROVED** [review]
+1

File: docs/artist-animation-clip.md (hunk 1 above)
Comment (luzhuang) [reply docs/artist-animation-clip.md]: fixed

**Pull request merged**